npm-shrinkwrap.json

```diff
@@ -8973,8 +8973,7 @@
     },
     "phone": {
       "version": "1.0.4-3",
-      "from": "git+https://github.com/Automattic/node-phone.git#0a3a87cc2b70906c7367ba3a9c9a27915f335443",
-      "resolved": "git+https://github.com/Automattic/node-phone.git#0a3a87cc2b70906c7367ba3a9c9a27915f335443"
+      "resolved": "git+https://github.com/Automattic/node-phone.git#8f012b153e968038c2ce758b7f34e5b8d792eb20"
```

Comment:
The resolved is optional here unless this is only available from a private registry